keylime/ima/ima.py

```diff
@@ -33,7 +33,6 @@
 EMPTY_ALLOWLIST = {
     "meta": {
         "version": ALLOWLIST_CURRENT_VERSION,
-        "checksum": "1ab36a57ad793caba756b01fc142f296cbfc3d24088fe6d66448f3bb6ebadeee",
```

Comment:
I think that makes sense to leave the field empty: `"checksum": "",`

This can help documentation (expected schema)

Reply:
So there is this tool, cmd/convert_ima_policy.py, that copy this `dict` and transform it. As today `checksum` is not touched, but seems to be that uses this as a template, and I guess that a correct transformation can touch this parameter in the future.

Reply:
We should not be putting `checksum` within the policy itself, IMO. Editing policies after creation to add the checksum in will break signing/verifying steps - it's crucial that the file provided to Keylime at the beginning remains the same, and is never modified.

If we want to add checksums, it should happen entirely on the verifier.

Reply:
@aplanas Because of the above reasoning, the IMA policy overhaul will not use the `checksum` field in the future.

Reply:
Yes, there are **two** `checksums`. Regrettably, my previous PR #1181 was conflating the two, but this is now fixed on this one. The checksum that matters is the one calculated **entirely on the verifier**

Reply:
So, if IMA policy overhaul stops using the `checksum` key within `meta`, it will **not** affect anything on this PR.